examples/awg.md

```diff
@@ -30,16 +30,30 @@ device. In zhinst-toolkit this module is also available. In addition the devices
 with AWG support also have helper function directly implemented in the node tree.
 
 The functionality explained in the following example are valid for all devices
-and are available behind the ``/awg`` node.
+and are available behind the ``awg`` node.
 
-Connect devices and access the ``/awg`` node.
+Connect devices and access the ``awg`` node.
 
 ```python
+# Load the LabOne API and other necessary packages
 from zhinst.toolkit import Session
 
-session = Session("localhost")
-device = session.connect_device("DEVXXXX")
-awg_node = device.awgs[0]
+DEVICE_ID = 'DEVXXXXX'
+SERVER_HOST = 'localhost'
+AWG_CORE = 0
+
+## connect to data server
+session = Session(SERVER_HOST)
+
+## connect to device
+device = session.connect_device(DEVICE_ID)
```

Comment:
```
session = Session("localhost")
device = session.connect_device("DEVXXXX")
```

This is mandatory for now for each example (to run them within regression infrastructure)